.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,19 @@
+name: pull request jobs
+
+on:
+  push:
+    branches-ignore:
+      - 'main'
+  workflow_dispatch:
+  workflow_call:
```

Comment:
Drop workflow_dispatch and workflow_call as there should be no reason to call this manually or in another workflow.